### PR TITLE
Filtered cards

### PR DESCRIPTION
**Pull request by Masuda50**
After the cards values are calculated they are filtered according to the values input on the form.

### CODE REVIEW

File: cards/views.py [hunk 1]
```diff
@@ -118,6 +131,42 @@ def calculate_card_value(card, grocery_input, dining_out_input, gas_input, trave
...
+    else: 
+        for i in f1:
+            f2.append(i)
+
+    for k in f2:
+        if k.bankName in banks_input:
+            f3.append(k)
+
+    return f3;
+
```
Comment (natharrow) [inline cards/views.py:169]: What do f1, f2 and f3 stand for? Maybe we could change those to more descriptive names?

Comment (Quetourah) [reply cards/views.py:169]: I agree

File: cards/views.py [hunk 2]
```diff
@@ -36,19 +36,27 @@ def get_info(request):
             annual = form.cleaned_data['annual']
             banks = form.cleaned_data['banks']
             if banks == ['all']:
-                banks = ['chase', 'citi', 'amex', 'capital one', 'bank of america', 'wells fargo']
+                banks = ['Chase', 'Citibank', 'American Express', 'Capital One', 'Bank of America', 'Wells Fargo']
```
Comment (Quetourah) [inline cards/views.py:39]: This list might get really big. Is there a simple way to not have the values hardcoded?

File: cards/views.py [hunk 3]
```diff
@@ -118,6 +131,42 @@ def calculate_card_value(card, grocery_input, dining_out_input, gas_input, trave
...
+    elif credit_score_input == 'good':
+        for i in f1:
+            if i.creditScore == "Bad" or i.creditScore == "Average" or i.creditScore == "Good":
+                f2.append(i)
+    else: 
+        for i in f1:
+            f2.append(i)
+
+    for k in f2:
+        if k.bankName in banks_input:
```
Comment (Quetourah) [inline cards/views.py:165]: Julia had mentioned using a consistent naming format for our variables according to python's `under_score` format. `bankName` and `creditScore` could be changed to that format.

**Pull request merged**